hawc/apps/lit/templates/lit/fragments/workflow_edit_row.html

```diff
@@ -62,19 +62,21 @@ <h3 class="py-2">Create Workflow</h3>
             <button class="btn btn-light"
                     id="workflow-cancel"
                     type='button'
-                    onclick="$(this).closest('.workflow-edit-row').remove()"
+                    onclick="delRow($(this).closest('.workflow-edit-row'))"
                     style="width: 10rem; height: 2rem;">
               Cancel
             </button>
           {% endif %}
         </div>
       </form>
+      {% include "common/helptext_popup_js.html" %}
+    </div>
   {% endif %}
-</div>
-</div>
-{% if form %}
-  <script type="text/javascript">
-    document.querySelector('#id_title').focus();
+  <script>
+    function delRow(elt) {
+      var scrollVal = $(elt).attr('scrolltop');
```

Comment:
can this be a helper method too? honestly I think we'd want this version of a deletion everywhere